pvlib/iam.py

```diff
@@ -863,6 +874,35 @@ def schlick_diffuse(surface_tilt):
     iam_ground : numeric
         The incident angle modifier for ground-reflected diffuse.
 
+    See Also
+    --------
+    pvlib.iam.schlick
+
+    Notes
+    -----
+    The analytical integration of the Schlick approximation was derived
+    as part of the FEDIS diffuse IAM model [2]_.  Compared with the model
+    implemented in this function, the FEDIS model includes an additional term
+    to account for reflection off a pyranometer's glass dome.  Because that
+    reflection should already be accounted for in the instrument's calibration,
+    the pvlib authors believe it is incorrect to account for it again in an
+    IAM model.  Thus, this function omits that term and implements only
+    the integration of the Schlick approximation.
```

Comment:
```suggestion
    to account for reflection off a pyranometer's glass dome.  Because that
    reflection should already be accounted for in the instrument's calibration,
    the pvlib authors believe it is inappropriate to account for pyranometer
    reflection again in an IAM model.  Thus, this function omits that term and
    implements only the integrated Schlick approximation.
```